agent/agentscripts/agentscripts_windows.go

```diff
@@ -1,7 +1,16 @@
 package agentscripts
 
-import "syscall"
+import (
+	"os/exec"
+	"syscall"
+)
 
 func cmdSysProcAttr() *syscall.SysProcAttr {
 	return &syscall.SysProcAttr{}
 }
+
+func cmdCancel(cmd *exec.Cmd) func() error {
+	return func() error {
+		return nil
```

Comment:
Could we use `cmd.Process.Signal(os.Interrupt)` here? I wonder if a non-nil cancel function will just prevent any interrupt from being sent?

Reply:
True. I will fix!